README.md

```diff
@@ -1,18 +1,18 @@
 # Lovelace Valetudo Map Card
 
-Draws the map available from a Xiaomi Vacuum cleaner flashed with [Valetudo](https://github.com/Hypfer/Valetudo) in a [Home Assistant](https://www.home-assistant.io/) Lovelace card.
+Draws the map that is provided by [ICantBelieveItsNotValetudo](https://github.com/Hypfer/ICantBelieveItsNotValetudo) from a Xiaomi vacuum cleaner, that is rooted and flashed with [Valetudo](https://github.com/Hypfer/Valetudo) in a [Home Assistant](https://www.home-assistant.io/) Lovelace card.
```

Comment:
This component doesn't use ICantBelieveItsNotValetudo though, it draws the map itself

Reply:
Okay, I thought valetudo-map-card would use the map provided by ICantBelieveItsNotValetudo, but as it does not, I removed that phrase.